product_supplierinfo_for_customer/models/product_product.py

```diff
@@ -10,6 +10,125 @@
 class ProductProduct(models.Model):
     _inherit = 'product.product'
 
+    @api.multi
+    def name_get(self):
+
+        def _name_get(d):
```

Comment:
Don't do nested methods that are not inheritable

Reply:
but, it's the same as odoo method :S

see in https://github.com/OCA/OCB/blob/12.0/addons/product/models/product.py#L425

Reply:
That doesn't mean it should be the best way. And what I don't like is that you make a verbatim copy of everything. See the trick I propose above.